subworkflows/local/allele_specific_calling.nf

```diff
@@ -34,25 +40,75 @@ workflow ALLELE_SPECIFIC_CALLING {
         )
         ch_versions = ch_versions.mix(BCFTOOLS_INDEX.out.versions.first())
 
-        dict_no_meta = dict.map{ meta, it -> [it] }.collect()
+        ch_dict_no_meta = ch_dict.map{ meta, it -> [it] }.collect()
         GATK4_ASEREADCOUNTER(
-            bam_bai,
+            ch_bam_bai,
             BCFTOOLS_VIEW.out.vcf.join(BCFTOOLS_INDEX.out.tbi),
-            fasta,
-            fai,
-            dict_no_meta,
-            intervals
+            ch_fasta,
+            ch_fai,
+            ch_dict_no_meta,
+            ch_intervals
         )
         ch_versions = ch_versions.mix(GATK4_ASEREADCOUNTER.out.versions.first())
 
         BOOTSTRAPANN(
-            vcf_tbi,
+            ch_ind_vcf_tbi,
             GATK4_ASEREADCOUNTER.out.csv
         )
         ch_versions = ch_versions.mix(BOOTSTRAPANN.out.versions.first())
 
+        TABIX_BGZIP(BOOTSTRAPANN.out.vcf)
+        ch_versions = ch_versions.mix(TABIX_BGZIP.out.versions.first())
+
+        BCFTOOLS_INDEX_2(TABIX_BGZIP.out.output)
+        ch_versions = ch_versions.mix(BCFTOOLS_INDEX_2.out.versions.first())
```

Comment:
If you want to you can use the `TABIX_BGZIPTABIX` module to produce [ meta, vcf.gz, vcf.gz.tbi ] directly. That would however require an update to how the files are collected downstream. We can wait with this as well